docs/source/ProgrammingGuide/DeprecatedInitialization.md

```diff
@@ -0,0 +1,117 @@
+# 15. Deprecated Initialization: THIS PAGE IS VALID FOR KOKKOS 3.6 AND OLDER
+
+In order to use Kokkos an initialization call is required. That call is responsible for initializing internal objects and acquiring hardware resources such as threads. Typically, this call should be placed right at the start of a program. If you use both MPI and Kokkos, your program should initialize Kokkos right after calling `MPI_Init`. That way, if MPI sets up process binding masks, Kokkos will get that information and use it for best performance. Your program must also _finalize_ Kokkos when done using it in order to free hardware resources.
+
+## 15.0 Include Headers
+
+All primary capabilities of Kokkos are provided by the `Kokkos_Core.hpp` header file.
+Some capabilities - specifically data structures in the `containers` subpackage and algorithmic capabilities in the `algorithms` subpackage are included via separate header files.
+For specific capabilities check their API reference:
+- [API: Core](../API/core-index)
+- [API: Containers](../API/containers-index)
+- [API: Algorithms](../API/algorithms-index)
+- [API in Alphabetical Order](../API/alphabetical)
+
+## 15.1 Initialization by command-line arguments
+
+The simplest way to initialize Kokkos is by calling the following function:
+```c++
+Kokkos::initialize(int& argc, char* argv[]);
+```
+Just like `MPI_Init`, this function interprets command-line arguments to determine the requested settings. Also like `MPI_Init`, it reserves the right to remove command-line arguments from the input list. This is why it takes `argc` by reference, rather than by value; it may change the value on output.
+
+During initialization one or more execution spaces will be initialized and assigned to one of the following aliases.
+
+```c++
+Kokkos::DefaultExecutionSpace;
+```
+```c++
+Kokkos::DefaultHostExecutionSpace;
+```
+
+`DefaultExecutionSpace` is the execution space used with policies and views where one is not explicitly specified.  Primarily, Kokkos will initialize one of the heterogeneous backends (CUDA, OpenMPTarget, HIP, SYCL) as the `DefaultExecutionSpace` if enabled in the build configuration.  In addition, Kokkos requires a `DefaultHostExecutionSpace`.  The `DefaultHostExecutionSpace` is default execution space used when host operations are required.  If one of the parallel host execution spaces are enabled in the build environment then `Kokkos::Serial` is only initialized if it is explicitly enabled in the build configuration.  If a parallel host execution space is not enabled in the build configuration, then `Kokkos::Serial` is initialized as the `DefaultHostExecutionSpace`.
+Kokkos chooses the two spaces using the following list:
```

Comment:
Instead of "the two spaces," separate into two lists of host (CPU) execution spaces and device (GPU) execution spaces.